components/openzwave-adapter/src/lib.rs

```diff
@@ -241,10 +258,12 @@ impl OpenzwaveAdapter {
             setter_map: IdMap::new(),
             watchers: Arc::new(Mutex::new(Watchers::new())),
             value_cache: Arc::new(Mutex::new(HashMap::new())),
+            controller_map: IdMap::new(),
```

Comment:
I'm not using this one yet but we'll need it when we'll implement messages to remove a Controller.